frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/flavor.tsx

```diff
@@ -127,7 +130,7 @@ export const FlavorSelect: React.FC<FlavorProps> = React.memo(
                     {isDefault
                       ? t('kubevirt-plugin~{{flavor}} (default)', { flavor: t(f.toString()) })
                       : t(f.toString())}
```

Comment:
```suggestion
                      : flavorDesc }
```

Reply:
See my answer below about the difference between `f.toString()` and `flavorDesc`.